examples/camera-controller/webrtc_manager/WebRTCManager.cpp

```diff
@@ -89,8 +90,14 @@ CHIP_ERROR WebRTCManager::Connnect(Controller::DeviceCommissioner & commissioner
 
     rtc::InitLogger(rtc::LogLevel::Warning);
 
+    initializeSocket();
```

Comment:
Consider handling error conditions from this function and returning.

Reply:
Thank you for your feedback. We added